easyconfigs/e/eo-learn/eo-learn-1.5.7-foss-2023a.eb

```diff
@@ -0,0 +1,115 @@
+    ('Pygments', '2.18.0'),
+    ('numba', '0.58.1'),
+    ('protobuf-python', '4.24.0'),
+]
+
+sanity_pip_check = True
+use_pip = True
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+}
+
+exts_list = [
+    ('Jinja2', '3.1.2', {
+        'checksums': ['31351a702a408a9e7595a8fc6150fc3f43bb6bf7e319770cbc0db9df9437e852'],
+    }),
+    ('aenum', '3.1.15', {
+        'checksums': ['8cbd76cd18c4f870ff39b24284d3ea028fbe8731a58df3aa581e434c575b9559'],
+    }),
+    ('dunamai', '1.23.0', {
+        'checksums': ['a163746de7ea5acb6dacdab3a6ad621ebc612ed1e528aaa8beedb8887fccd2c4'],
+    }),
+    ('poetry_dynamic_versioning', '1.5.1', {
+        'checksums': ['6d6a626f32cd8034416d2dd690e81b21ee2da9e153bf69e5725a7976fbda6ae5'],
+    }),
+    ('marshmallow', '3.26.1', {
+        'checksums': ['e6d8affb6cb61d39d26402096dc0aee12d5a26d490a121f118d2e81dc0719dc6'],
+    }),
+    ('mypy_extensions', '1.0.0', {
+        'checksums': ['75dbf8955dc00442a438fc4d0666508a9a97b6bd41aa2f0ffe9d2f2725af0782'],
+    }),
+    ('typing_inspect', '0.9.0', {
+        'checksums': ['b23fc42ff6f6ef6954e4852c1fb512cdd18dbea03134f91f856a95ccc9461f78'],
+    }),
+    ('dataclasses_json', '0.6.7', {
+        'checksums': ['b6b3e528266ea45b9535223bc53ca645f5208833c29229e847b3f26a1cc55fc0'],
+    }),
+    ('oauthlib', '3.2.2', {
+        'checksums': ['9859c40929662bec5d64f34d01c99e093149682a3f38915dc0655d5a633dd918'],
+    }),
+    ('requests-oauthlib', '2.0.0', {
+        'checksums': ['b3dffaebd884d8cd778494369603a9e7b58d29111bf6b41bdc2dcd87203af4e9'],
+    }),
+    ('tifffile', '2025.1.10', {
+        'checksums': ['baaf0a3b87bf7ec375fa1537503353f70497eabe1bdde590f2e41cc0346e612f'],
+    }),
+    ('utm', '0.8.0', {
+        'checksums': ['37143e7bad8390ad974fdb66de3156128f798ac54f861c2cd1c931ab411568aa'],
+    }),
+    ('sentinelhub', '3.11.1', {
+        'checksums': ['559d03fa8a67255bcf92ac10347386e74e81ca9839b5eccf75b614bd747a752d'],
+    }),
+    ('aiosignal', '1.3.2', {
+        'checksums': ['a8c255c66fafb1e499c9351d0bf32ff2d8a0321595ebac3b93713656d2436f54'],
+    }),
+    ('expandvars', '0.12.0', {
+        'checksums': ['7d1adfa55728cf4b5d812ece3d087703faea953e0c0a1a78415de9df5024d844'],
+    }),
+    ('frozenlist', '1.5.0', {
+        'checksums': ['81d5af29e61b9c8348e876d442253723928dce6433e0e76cd925cd83f1b4b817'],
+    }),
+    ('ray', '2.41.0', {
+        'source_tmpl': '%(name)s-%(version)s-cp311-cp311-manylinux2014_%(arch)s.whl',
+        'checksums': ['fff5e9cc5a53815d3b586a261e34bd0fef1c324b2cded4c9b8e790e1e3dc3997'],
+    }),
+    ('fs', '2.4.16', {
+        'checksums': ['ae97c7d51213f4b70b6a958292530289090de3a7e15841e108fbe144f069d313'],
+    }),
+    ('fs-s3fs', '1.1.1', {
+        'checksums': ['b57f8c7664460ff7b451b4b44ca2ea9623a374d74e1284c2d5e6df499dc7976c'],
+    }),
+    ('eo_learn', version, {
+        'modulename': 'eolearn',
+        'preinstallopts': "sed -ie '/.*geopandas.*14.4.*/d; /.*opencv.*headless.*/d' pyproject.toml && ",
```

Comment:
With `ie` you're adding the flag `i` and `e` as an argument to create a backup. Is this desired? If you're just running a single command of in-file editing, "sed -i" should be sufficient?